interface.cpp

```diff
@@ -110,6 +120,11 @@ struct call_traits<'u'> {
         free(buf);
         return ret;
     }
+
+    // Convert std::strings to char*s
+    static const char* to_bytes(std::wstring& value) {
+        return (char*) value.c_str();
```

Comment:
Hi & thanks! This one doesn't seem right - you're casting from wchar_t* to char*.